plot/plot.py

```diff
@@ -9,6 +9,18 @@
 # pylint: disable-msg=R0912
 
 
+# def psd_helper(samples, sampling_freq, center_freq):
```

Comment:
Willen wij deze functie wel of niet in de master hebben?

Reply:
wellicht in de toekomst, haal het nu liever niet weg.